cric-exchange-app-server/src/index.ts

```diff
@@ -0,0 +1,14 @@
+import {createConnection} from "typeorm";
+import {User} from "./entity/User";
+
+// connection settings are in the "ormconfig.json" file
+createConnection().then(async connection => {
```

Comment:
How is this user creation called from outside this file?

Reply:
The scaffolding will come in a later MR.